cql3/expr/expression.cc

```diff
@@ -261,7 +261,7 @@ bool limits(const std::vector<column_value>& columns, const oper_t op, term& t,
     for (size_t i = 0; i < rhs.size(); ++i) {
         const auto cmp = get_value_comparator(columns[i])->compare(
                 // CQL dictates that columns[i] is a clustering column and non-null.
-                *get_value(columns[i], bag),
+                managed_bytes_view(*get_value(columns[i], bag)),
```

Comment:
Why is this necessary?  Doesn't `get_value` return a `managed_bytes_view` now?

Reply:
Right, it's not neccessary because the conversion from `managed_bytes` to `managed_bytes_view` can be implicit here. Removed.